### ISSUE TITLE
macOS Client

### ISSUE THREAD

**Issue by mrsaraiva** [ofeefee/fluttercoin#41]
Hello,

Are you going to release a macOS version of the latest client before the fork?

**Comment by slax0r** [comment]
Just submitted a pull request to take care of that.

**Comment by ofeefee** [comment]
Committed

**Comment by kizeren** [comment]
I do believe the brew install miniupnpc will not work as it will install the latest version which is broken for most coins.  Also the brew install openssl installs the newest version which is also broken for some coins.

**Comment by slax0r** [comment]
Yep it does, except I patched the only file it breaks to work with the new version.

**Comment by slax0r** [comment]
OpenSSL doesnt seem to be broken for Flutter. Ive got it running and syncing now with no issues at all.
